src/dom.ts

```diff
@@ -34,6 +78,30 @@ export function byId(id: string): HTMLElement {
 }
 
 /**
+ * Determines whether an ELement has a CSS class name
+ *
+ * @param element The Element to which to check for a CSS class
+ * @param className The CSS class name to check for
+ *
+ * @example
+ * var loaded = dom.containsClass(document.body, 'loaded');
+ */
+export function containsClass(element: Element, className: string): boolean {
+	let targetElement = <any> element;
+	if (!targetElement) {
+		return;
+	}
+	if (targetElement.classList) {
+		let classList: any = targetElement.classList;
+		return classList.contains(className);
+	}
+	else {
+		validateClass(className);
+		return targetElement.className.indexOf(className) > -1;
```

Comment:
This is overly naive.  For an element with a `className` of `foobar`, this would return true for `containsClass(element, 'foo')`.

I'd suggest adopting Dojo 1's implementation for this one.